broker/src/main/java/org/apache/rocketmq/broker/config/v2/SubscriptionGroupManagerV2.java

```diff
@@ -0,0 +1,173 @@
+
+        try (RocksIterator iterator = configStorage.iterate(beginKey.nioBuffer(), endKey.nioBuffer())) {
+            while (iterator.isValid()) {
+                SubscriptionGroupConfig subscriptionGroupConfig = parseSubscription(iterator.key(), iterator.value());
+                if (null != subscriptionGroupConfig) {
+                    super.updateSubscriptionGroupConfigWithoutPersist(subscriptionGroupConfig);
+                }
+            }
+        } finally {
+            beginKey.release();
+            endKey.release();
+        }
+        return true;
+    }
+
+    private SubscriptionGroupConfig parseSubscription(byte[] key, byte[] value) {
+        ByteBuf keyBuf = Unpooled.wrappedBuffer(key);
+        ByteBuf valueBuf = Unpooled.wrappedBuffer(value);
+        try {
+            // Skip table-prefix, table-id, record-type-prefix
+            keyBuf.readerIndex(4);
+            short groupNameLen = keyBuf.readShort();
+            assert groupNameLen == keyBuf.readableBytes();
+            CharSequence groupName = keyBuf.readCharSequence(groupNameLen, StandardCharsets.UTF_8);
+            assert null != groupName;
+            byte serializationType = valueBuf.readByte();
+            if (SerializationType.JSON == SerializationType.valueOf(serializationType)) {
+                CharSequence json = valueBuf.readCharSequence(valueBuf.readableBytes(), StandardCharsets.UTF_8);
+                SubscriptionGroupConfig subscriptionGroupConfig = JSON.parseObject(json.toString(), SubscriptionGroupConfig.class);
+                assert subscriptionGroupConfig != null;
+                assert groupName.equals(subscriptionGroupConfig.getGroupName());
+                return subscriptionGroupConfig;
+            }
+        } finally {
+            keyBuf.release();
+            valueBuf.release();
+        }
+        return null;
+    }
+
+    @Override
+    public synchronized void persist() {
+        try {
+            configStorage.flushWAL();
+        } catch (RocksDBException e) {
+            log.error("Failed to flush RocksDB WAL", e);
+        }
+    }
+
+    @Override
+    public SubscriptionGroupConfig findSubscriptionGroupConfig(final String group) {
+        if (MixAll.isLmq(group)) {
+            SubscriptionGroupConfig subscriptionGroupConfig = new SubscriptionGroupConfig();
+            subscriptionGroupConfig.setGroupName(group);
+            return subscriptionGroupConfig;
+        }
+        return super.findSubscriptionGroupConfig(group);
+    }
+
+    @Override
+    public void updateSubscriptionGroupConfig(final SubscriptionGroupConfig config) {
+        if (config == null || MixAll.isLmq(config.getGroupName())) {
+            return;
+        }
+        ByteBuf keyBuf = ConfigHelper.keyBufOf(TableId.SUBSCRIPTION_GROUP, config.getGroupName());
+        ByteBuf valueBuf = ConfigHelper.valueBufOf(config, SerializationType.JSON);
+        try {
+            WriteBatch writeBatch = new WriteBatch();
```

Comment:
Need to release the wirteBatch underlying resoueces by calling close() or using try-with-resources.

Reply:
Good catch